templates/zerver/landing_nav.html

```diff
@@ -72,6 +72,11 @@
                                 Managing hundreds of projects at End Point Dev
                                 </a>
                             </li>
+                            <li class="top-menu-submenu-list-item">
+                                <a href="https://zulip.com/case-studies/semsee/">
```

Comment:
In general you should just start with these being relative links, and we can adjust them in follow-up passes, so that CI passes. The benefit of the `zulip.com` URL over the relative one is pretty low.

Reply:
All the other landing nav links are absolute, so I don't think that makes sense to do just for this one?

Reply:
Oh, I see, you're saying we can merge this with a relative link, and then fix it later? OK.